easybuild/easyconfigs/c/cutadapt/cutadapt-1.8.1-goolf-1.7.20-Python-2.7.9.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = "PythonPackage"
+
+name = 'cutadapt'
+version = '1.8.1'
+
+homepage = 'http://code.google.com/p/cutadapt/'
+description = """ cutadapt removes adapter sequences from high-throughput sequencing data. 
+ This is usually necessary when the read length of the sequencing machine is longer than 
+ the molecule that is sequenced, for example when sequencing microRNAs.  """
+
+toolchain = {'name': 'goolf', 'version': '1.7.20'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
+
+python = 'Python'
+pyver = '2.7.9'
+pyshortver = '.'.join(pyver.split('.')[:2])
+
+versionsuffix = "-%s-%s" % (python, pyver)
+
+dependencies = [
+    (python, pyver),
+]
+
+sanity_check_paths = {
+    'files': ['bin/cutadapt','lib/python%s/site-packages/cutadapt/_align.so' % pyshortver],
```

Comment:
shouldn't be using hardcoded `.so`, but `.../_align.%s' % (pyshortver, SHLIB_EXT)` instead